lib/functions.sh

```diff
@@ -314,23 +314,23 @@ function get_or_create_role {
 function get_or_add_user_project_role {
     local user_role_id
     # Gets user role id
-    user_role_id=$(openstack role list \
+    user_role_id=$(openstack role assignment list \
         --user $2 \
-        --column "ID" \
         --project $3 \
-        --column "Name" \
-        | grep " $1 " | get_field 1)
+        --column "Role" \
+        --format "value" \
+        | grep "$1")
```

Comment:
The command doesn't work as previously. I assume you should replace `| grep "$1"` with `--role $1`.

Reply:
Yes, thank you